pnpm-lock.yaml

```diff
@@ -551,7 +549,7 @@ importers:
         version: 2.0.3
       '@wdio/cli':
         specifier: ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0
-        version: link:../wdio-cli
```

Comment:
I don't think this is correct, I will have to check whether dependabot requires different setting for PNPM

Reply:
Ah, OK. I'll have a look.